packages/contracts-bedrock/deployPeptideL1Contracts.sh

```diff
@@ -0,0 +1,20 @@
+
+export OLD_DEPLOY_CONFIG_PATH=./deploy-config/polymer-mainnet.json
+export DEPLOY_CONFIG_PATH=./deploy-config/polymer-mainnet-1.json
+export IMPL_SALT="polymer-deploy-1"
+
+cat $DEPLOY_CONFIG_PATH
+
+export blockNumber=$(cast block --rpc-url $LOCAL_RPC| grep "number" | grep -Eo '[0-9]+' | sed 's/\.$//')
```

Comment:
can you use '-j' option to print the cast result as json and then use 'jq' to get the number?

Reply:
ahh good one, thanks for the suggestion!

Reply:
should be updated now.  -j returns the format in hex so we have to do a cast to-dec to convert this (though lmk if there's a way to do this in jq natively?)